.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: node_js
 node_js:
 - 6
+- 8
 script:
 - npm run build
+before_install:
+- npm install -g npm@latest
 after_success:
 - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "master" && bash
   deploy.sh
+cache: false
 env:
```

Comment:
since we are in here and NOT actively debugging anymore, we should be able to nix this whole env>global>secure nonsense and just make sure that env var is set in the travis dashboard